src/harness/testcases/WINNF_FT_S_PCR_testcase.py

```diff
@@ -0,0 +1,643 @@
+    device_b['installationParam']['latitude'], device_b['installationParam'][
+        'longitude'] = 39.0378, -100.4785
+    device_c['installationParam']['latitude'], device_c['installationParam'][
+        'longitude'] = 39.0378, -100.4463
+
+    # Set the AntennaGain and EIRP capability same CBSD detail PCR.1 is used.
+    device_a['installationParam']['eirpCapability'] = 30
+    device_b['installationParam']['eirpCapability'] = 47
+    device_a['installationParam']['antennaGain'] = 16
+    device_b['installationParam']['antennaGain'] = 16
+
+    conditionals_b = {
+        'cbsdCategory': device_b['cbsdCategory'],
+        'fccId': device_b['fccId'],
+        'cbsdSerialNumber': device_b['cbsdSerialNumber'],
+        'airInterface': device_b['airInterface'],
+        'installationParam': device_b['installationParam'],
+        'measCapability': device_b['measCapability']
+    }
+    conditionals = [conditionals_b]
+    del device_b['installationParam']
+    del device_b['cbsdCategory']
+    del device_b['airInterface']
+    del device_b['measCapability']
+
+    # Create the actual config.
+    devices = [device_a, device_b, device_c]
+    config = {
+        'registrationRequests': devices,
+        'conditionalRegistrationData': conditionals,
+        'palRecords': pal_records,
+        'sasTestHarnessCert': os.path.join('certs', 'sas.cert'),
+        'sasTestHarnessKey': os.path.join('certs', 'sas.key'),
+        'userClaimedPpaContour': user_claimed_ppa_contour_geometry
+    }
+    writeConfig(filename, config)
+
+  @configurable_testcase(generate_PCR_3_default_config)
+  def test_WINNF_FT_S_PCR_3(self, config_filename):
+    """Successful PPA Confirmation with Claimed Boundary by PAL Holder.
+
+    Checks PPA generated by SAS UUT shall be fully contained within the service area.
+    Checks SAS UUT shall confirm a valid PPA boundary claimed by the PAL holder,
+    composed of one or more adjacent Census Tracts.
+    """
+    # Load the Config file.
+    config = loadConfig(config_filename)
+
+    # Inject the PAL records.
+    for pal_record in config['palRecords']:
+        self._sas_admin.InjectPalDatabaseRecord(pal_record)
+
+    # Register devices and check the response.
+    cbsd_ids = self.assertRegistered(config['registrationRequests'],
+                                     config['conditionalRegistrationData'])
+
+    # Trigger SAS UUT to create a PPA boundary.
+    pal_ids = [record['palId'] for record in config['palRecords']]
+
+    # Create PPA creation request with user claimed ppa contour.
+    ppa_creation_request = {
+        "cbsdIds": cbsd_ids,
+        "palIds": pal_ids,
+        "providedContour": config['userClaimedPpaContour']
+    }
+
+    # Trigger PPA Creation to SAS UUT.
+    ppa_id = self.triggerPpaCreationAndWaitUntilComplete(ppa_creation_request)
+    logging.debug('ppa_id received from SAS UUT:%s', ppa_id)
+
+    # Notify SAS UUT about SAS Harness and trigger Full Activity Dump and retrieves the
```

Comment:
You haven't done the first part of this (notify).

Reply:
Notify SAS UUT via injectPeer SAS admin API is already done inside the method triggerFadGenerationAndRetrievePpaZone . So avoid the confusion we have repharse the comment line accordingly